packages/block-editor/src/components/rich-text/index.native.js

```diff
@@ -59,6 +59,14 @@ import EmbedHandlerPicker from './embed-handler-picker';
 
 const classes = 'block-editor-rich-text__editable';
 
+function limitIntegerValue( value ) {
```

Comment:
Would it be tricky to make this reusable so we don't have to define two similar functions?

Reply:
Good question. I'm not sure we have a common native util lib but I'll check. I don't think, however, it would be worth creating one in this situation.

Reply:
I looked into this and the problem is that new function is used by components in different packages: `block-editor` and `block-library`. To DRY it up it would need to be in a third package and I don't see a good existing candidate package to host this function.  
There are some options to place the function in a more _utility_ specific file within each package but that doesn't really solve much. At least while the function is only used in one instance in each package.

Reply:
Sounds good. 👍